spec/views/hyrax/dashboard/_sidebar.html.erb_spec.rb

```diff
@@ -107,4 +107,15 @@
     it { is_expected.to have_content t('hyrax.admin.sidebar.configuration') }
     it { is_expected.to have_link t('hyrax.admin.sidebar.collection_types') }
   end
+
+  context 'when proxy deposits are enabled' do
+    before do
+      allow(Flipflop).to receive(:proxy_deposit?).and_return(true)
+      render
+    end
+
+    subject { rendered }
+
+    it { is_expected.to have_link t('hyrax.dashboard.manage_proxies') }
+  end
```

Comment:
I'm not seeing the disabled version of this test.  Is it somewhere else?  I saw where it was removed from profiles/edit.